client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,36 @@
+// HelloWorldWidget is an arbitrary name for any "dumb" component. We do not recommend suffixing
+// all your dumb component names with Widget.
+
+import React, { PropTypes } from 'react';
+
+// Simple example of a React "dumb" component
+const HelloWorldWidget = (props) => {
+  const { name, updateName } = props;
```

Comment:
I would either just use `props.name` and `props.updateName`, or I would destructure the props in the argument. This way you can avoid having to the return statement and such:


```js
const HelloWorldWidget = ({ name, updateName }) => (
  ...
);
```